lib/system-checker.coffee

```diff
@@ -25,20 +27,22 @@ class SystemChecker
   isEnabled: -> env.isSystemSupported()
   getStatus: ->
     if env.isSystemSupported()
-      "Working correctly"
+      "working correctly"
     else
-      "Disabled on Linux"
+      "disabled on Linux"
 
   providesSpelling: (args) -> @isEnabled()
   providesSuggestions: (args) -> @isEnabled()
   providesAdding: (args) -> false # Users can't add yet.
 
   check: (args, text) ->
     id = @getId()
+
     if @isEnabled()
       # We use the default checker here and not the locale-specific one so it
       # will check all languages at the same time.
-      instance.checkSpellingAsync(text).then (incorrect) ->
+      instance.checkSpellingAsync(text).then (incorrect) =>
+        @log 'check', text, incorrect
```

Comment:
Should probably remove this after debugging this PR, even with logging disabled, its needless overhead?

Reply:
I would like to keep this in. This is the third time I've had to create a branch that adds logging to figure out problems. However, I did have it do a check to see if logging is enabled and removed the text display, so that *should* reduce the performance hit significantly while checking but still give the ability to change it on the fly.